jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -424,7 +424,8 @@ def single_pipeline(job) {
         if (job_name == "puppetserver-infinite") {
 
             dir('simulation-runner/config/scenarios') {
-                job["gatling_simulation_config"] = generate_gatling_scenario(NUMBER_OF_HOURS, CATALOG_SIZE, NODE_COUNT)
+                scenario_name = generate_gatling_scenario(NUMBER_OF_HOURS, CATALOG_SIZE, NODE_COUNT)
+                job["gatling_simulation_config"] = "../simulation-runner/config/scenarios/${scenario_name}"
```

Comment:
The double dots are weird. I'm guessing that this assumes you're reading this variable from a specific directory? I think that should be the responsibility of the caller. ie no `"../"` on this variable.

Reply:
I don't disagree with the sentiment, but it is "../" in *all* of the Jenkinsfiles so I'm going for consistency.

Reply:
(for example https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-5.x/Jenkinsfile#L8)

Reply:
Eh, seems fine then I guess.

Reply:
Hrmm... that sucks. I guess we're at least consistent about always considering this data to be executed within the `jenkins-integration` sub directory of the repo: https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-5.x/Jenkinsfile#L21